frontend/src/metabase/home/homepage/components/HomeLayout/HomeLayout.styled.tsx

```diff
@@ -24,8 +29,20 @@ const gradientStyles = css`
 
 export const LayoutRoot = styled.div<LayoutProps>`
   min-height: 100%;
-  padding: 4rem 7rem;
+  padding: 1.5rem 1.5rem;
   ${props => (props.showScene ? sceneStyles : gradientStyles)};
+
+  ${breakpointMinMedium} {
+    padding: 2rem 3rem 2rem;
```

Comment:
@mazameli there is no design for the medium screen size in Figma, I just made up these numbers

Reply:
Thanks, yeah, I only specced the large size. Let me play around with this a bit more and see how it feels.

Reply:
Pushed a small tweak to this.